debs/focal/bulk_extractor/Makefile

```diff
@@ -0,0 +1,46 @@
+NAME          = bulk_extractor
+VERSION       ?= 2.0.0
+RELEASE	      ?= 1~20.04
+GIT_URL	      = https://github.com/simsong/bulk_extractor
+GIT_BRANCH    = 2.0.0
+DEB_TOPDIR    = "/debbuild"
+DOCKER_VOLUME = "/src"
+DOCKER_IMAGE  = "debbuild-$(NAME)-$(VERSION)"
+
+.PHONY: build-docker-image build deb-build deb-clean deb-test update-changelog
+
+all: build-docker-image build
+
+build-docker-image:
+	@echo "==> Building Docker image with build environment."
+	@docker build --tag "$(DOCKER_IMAGE)" --build-arg GIT_URL="$(GIT_URL)" --build-arg GIT_BRANCH=v"$(GIT_BRANCH)" --build-arg PACKAGE="$(NAME)" .
+
+update-changelog:
+	@echo "==> Update changelog."
+	@dch -v $(VERSION)-$(RELEASE) -D jammy "New upstream release
```

Comment:
Since this is the `debs/focal` directory, should these `jammy`s be updated to `focal`?

Reply:
I changed Makefiles for focal and jammy packages. I added a new variable "DISTRO_CODENAME" that makes it easier.